.github/workflows/e2e-test-no-domain.yaml

```diff
@@ -28,6 +28,11 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Pin Kubectl version
+        uses: azure/setup-kubectl@v2.0
```

Comment:
nit: looks like it's provided by azure, do we have counterpart provided by aws/kubernetes?